lib/test_result.dart

```diff
@@ -75,6 +76,88 @@ class TestResult {
     return TestAttachment.fromJson(response);
   }
 
+  static Future<TestResults> getTestResults({
+    required int testId,
+    int? limit,
+    int? offset,
+  }) async {
+    final url = '/get_results/$testId';
+
+    final queryParameters = <String, dynamic>{
+      'limit': limit,
+      'offset': offset,
+    };
+
+    return _performRequest(
+      requestUrl: url,
+      queryParameters: queryParameters,
+    );
+  }
+
+  static Future<TestResults> getCaseResults({
+    required int runId,
+    required int caseId,
```

Comment:
I like the way you named this method much better
```suggestion
  static Future<TestResults> getCaseResults(int caseId, {
    required int runId,
```